backend/api/v1/endpoints/chatbot/huggingface.py

```
@@ -86,14 +66,21 @@
         return answers[-1][0].strip()
     return text.strip()
 
+# -------------------- Endpoints --------------------
 
 @router.post("/upload_pdf/", response_model=BotResponse)
 async def upload_pdf(file: UploadFile, current_user: User = Depends(get_current_user)):
```

Ruff (0.8.2)

Missing return type annotation for public function `upload_pdf`

(ANN201)